packages/stac/lib/src/parsers/widgets/stac_backdrop_filter/stac_backdrop_filter_parser.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:flutter/material.dart';
+import 'package:stac/src/framework/framework.dart';
+import 'package:stac/src/parsers/widgets/stac_image_filter/stac_image_filter_parser.dart';
+import 'package:stac/src/utils/widget_type.dart';
+import 'package:stac_framework/stac_framework.dart';
+
+import 'stac_backdrop_filter.dart';
+
+class StacBackdropFilterParser extends StacParser<StacBackdropFilter> {
+  const StacBackdropFilterParser();
+
+  @override
+  String get type => WidgetType.backdropFilter.name;
+
+  @override
+  StacBackdropFilter getModel(Map<String, dynamic> json) =>
+      StacBackdropFilter.fromJson(json);
+
+  @override
+  Widget parse(BuildContext context, StacBackdropFilter model) {
+    final child = Stac.fromJson(model.child, context) ?? const SizedBox();
+
+    if (!model.enabled) {
+      return child;
+    }
+
+    final imageFilter = StacImageFilterParser.parse(model.filter);
+    if (imageFilter == null) {
+      return child;
+    }
+
+    return BackdropFilter(
+      filter: imageFilter,
+      child: child,
+    );
+  }
```

Comment:
We should throw error when imageFilter is missing. So, ideally we should remove the null check for imageFilter.